amethyst_renderer/Cargo.toml

```diff
@@ -58,6 +58,9 @@ gfx_window_dxgi = { version = "0.7", optional = true }
 [dependencies.hetseq]
 version = "0.1.5"
 
+[dependencies.smallvec]
```

Comment:
Why aren't these in the `dependencies` table?

Reply:
There was a feature. Now it's gone 😄